packages/client-sdk-web/src/internal/cache-data-client.ts

```diff
@@ -340,6 +355,8 @@ export class CacheDataClient<
     });
   }
 
+  // setIfNotExists is deprecated on the service. Here we call the new `SetIf` method with the absent field set
```

Comment:
I don't see the corresponding change for calling SetIfAbsent in sendSetIfNotExists?